test/libsolidity/syntaxTests/license/license_double5.sol

```diff
@@ -0,0 +1,4 @@
+/* SPDX-License-Identifier: GPL-3.0 OR GPL-2.0 */ /* SPDX-License-Identifier: GPL-3.0 OR GPL-2.0 */
+contract C {}
+// ----
+// ParserError 1114: Invalid SPDX license identifier.
```

Comment:
Since this is invalid, it probably sees `GPL-3.0 OR GPL-2.0 */ /* SPDX-License-Identifier: GPL-3.0 OR GPL-2.0` as the license name? I think that having something after the comment should be legal. I could imagine someone doing something like this:
```solidity
/* SPDX-License-Identifier: GPL-3.0 OR GPL-2.0 */ pragma solidity >=0.0;
```

Reply:
Also, this shows that it would really be nice to have a location for the error. Maybe we could at least include the invalid identifer in the error message? On the other hand if it contains invalid UTF-8 it might be a problem...

Reply:
You are right, this is a completely wrong test result. I had to fix the regex again to make this properly work, and it's so much better now. Good call!